Client/app/src/main/java/ru/hse/client/groups/GroupActivity.kt

```diff
@@ -2,46 +2,135 @@ package ru.hse.client.groups
 
 import android.content.Intent
 import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
 import android.util.Log
+import android.widget.AdapterView
+import android.widget.Toast
+import okhttp3.OkHttpClient
+import ru.hse.client.R
 import ru.hse.client.databinding.ActivityGroupBinding
 import ru.hse.client.utility.DrawerBaseActivity
 import ru.hse.server.proto.EntitiesProto
 import ru.hse.client.chapters.ChapterUploadActivity
+import ru.hse.client.chapters.ReadingChapterActivity
+import ru.hse.client.utility.user
 
 class GroupActivity: DrawerBaseActivity() {
 
     private lateinit var binding: ActivityGroupBinding
+    var group: EntitiesProto.GroupModel? = null
+    var chapters: EntitiesProto.ChapterList? = null
```

Comment:
it shouldn't be a Group activity field, it's used only as local variable in drawChapterList